docs/architecture/grpc-for-wcf-developers/self-hosted.md

```diff
@@ -43,19 +46,21 @@ To publish a framework-dependent build, omit the `-r` flag.
 dotnet publish -c Release -o ./publish
 ```
 
-Copy the complete contents of the `publish` directory to an installation folder, and use the [sc utility](https://docs.microsoft.com/windows/desktop/services/controlling-a-service-using-sc) to create a Windows service for the executable.
+Copy the complete contents of the `publish` directory to an installation folder. Then, use the [sc tool](/windows/desktop/services/controlling-a-service-using-sc) to create a Windows service for the executable file.
 
 ```console
 sc create MyService binPath=C:\MyService\MyService.exe
 ```
 
-### Log to Windows Event Log
+### Log to the Windows event log
+
+The `UseWindowsService` method automatically adds a [logging](/aspnet/core/fundamentals/logging/) provider that writes log messages to the Windows event log. You can configure logging for this provider by adding an `EventLog` entry to the `Logging` section of `appsettings.json` or another configuration source. 
 
-The `UseWindowsService` method automatically adds a [Logging](https://docs.microsoft.com/aspnet/core/fundamentals/logging/?view=aspnetcore-3.0) provider that writes log messages to the Windows Event Log. You can configure logging for this provider by adding an `EventLog` entry to the `Logging` section of `appsettings.json` or other configuration source. The source name used in Event Log can be overridden by setting a `SourceName` property in these settings; if you don't specify a name, the default application name (normally the executable assembly name) will be used.
+You can override the source name used in the event log by setting a `SourceName` property in these settings. If you don't specify a name, the default application name (normally the executable assembly name) will be used.
 
 More information on logging is at the end of this chapter.
 
-## Run your app as a Linux service with systemd
+## Run your app as a Linux service by using systemd
```

Comment:
I prefer the original phrasing here

Reply:
I made that change because of this entry in the Microsoft Writing Style Guide: https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=36373. Is "by using" technically inaccurate?

Reply:
But that entry also says it's ok to use with product names?

Reply:
True--though that's the not the case here. I can change the wording back to "with" if you feel strongly about it.

Reply:
@shirhatti?

Reply:
I changed it back to "with."